README.md

```diff
@@ -1,75 +1,93 @@
-Rundeck
-========
+<p align="center">
+<a href="https://www.rundeck.com#gh-light-mode-only">
+  <img src="https://www.rundeck.com/hubfs/Pager%20Duty%20Branding/RundeckbyPagerDuty.svg" alt="Netdata" width="300"/>
+</a>
+<a href="https://www.rundeck.com#gh-dark-mode-only">
+  <img src="https://www.rundeck.com/hubfs/Pager%20Duty%20Branding/RundeckbyPagerDutyDM.svg" alt="Netdata" width="300"/>
+</a>
+</p>
+<h3 align="center">Execute workflows across your existing automations<br /> or quickly automate previously manual procedures.</h3>
 
-| Travis | Deb | RPM | War |
-|--------|-----|-----|-----|
-|[![Travis CI](https://travis-ci.org/rundeck/rundeck.svg?branch=master)](https://travis-ci.org/rundeck/rundeck/builds#)|[Download](https://www.rundeck.com/downloads)|[Download](https://www.rundeck.com/downloads)|[Download](https://www.rundeck.com/downloads)|
+<br />
+<p align="center">
+<a href="https://github.com/rundeck/rundeck/"><img src="https://img.shields.io/github/stars/rundeck/rundeck?style=social" alt="GitHub Stars"></a>
+<a href="https://github.com/rundeck/rundeck/releases/latest"><img src="https://img.shields.io/github/release/rundeck/rundeck.svg" alt="Latest release"></a>
 
-Rundeck is an open source automation service with a web console,
-command line tools and a WebAPI.
-It lets you easily run automation tasks across a set of nodes.
+<div align="center">
 
-* Site: <https://www.rundeck.com>
+| Travis                                                                                                                 | Deb                                           | RPM                                           | War                                           |
+| ---------------------------------------------------------------------------------------------------------------------- | --------------------------------------------- | --------------------------------------------- | --------------------------------------------- |
+| [![Travis CI](https://travis-ci.org/rundeck/rundeck.svg?branch=master)](https://travis-ci.org/rundeck/rundeck/builds#) | [Download](https://www.rundeck.com/downloads) | [Download](https://www.rundeck.com/downloads) | [Download](https://www.rundeck.com/downloads) |
```

Comment:
you can remove Travis, we don't use that anymore

Reply:
Replaced Travis CI badge with Github Actions workflow for Create Release badge.